trinity/sync/beam/chain.py

```diff
@@ -755,6 +766,25 @@ async def import_block(
             f'{block.header.gas_used:,d}',
         )
 
+        if not isinstance(self.metrics_registry, NoopMetricsRegistry):
+            wit_db = AsyncWitnessDB(self._db)
+            try:
+                wit_hashes = wit_db.get_witness_hashes(block.hash)
+            except WitnessHashesUnavailable:
+                self.logger.debug(
+                    "No witness hashes for block %s. Import will be slow",
+                    encode_hex(block.hash),
+                )
+            else:
+                missing = self._state_downloader._get_unique_missing_hashes(wit_hashes)
+                self.logger.debug(
+                    "Missing %d witnesses out of %d for block %s",
```

Comment:
The combined thing is the witness. Maybe something like:
```suggestion
                    "Missing %d nodes from witness of %d for block %s",
```

Reply:
Went with `Missing %d nodes out of %d from witness of block %s`